Index.html

```diff
@@ -0,0 +1,71 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="stylesheet" href="style.css" />
+    <title>Mobile First Hussen Portfolio</title>
+  </head>
+  <body>
+    <main class="main-container">
+      <header class="top-bar">
+        <div class="tool-bar">
+          <div class="logo-setting">
+            <div class="logo">
+              <a href="#"><img src="img/My-Logo.png" alt="HY" /></a>
+              <!-- <img src="img/Hu-Photo-2-3.jpeg" alt="HY" width="76px" height="20px"> -->
+            </div>
+          </div>
+          <nav class="dash-setting">
+            <div class="icons">
+              <div class="icon-Union">
+                <img src="img/Union.png" alt="Three Dashs" />
+              </div>
+            </div>
+          </nav>
+        </div>
+      </header>
+      <section class="content-section">
+        <div class="bg-shapes">
+          <div class="text-1">
+            <h2>I’m Hussen Glad to see you!</h2>
+          </div>
+          <div class="text-2">
+            <p>
+              I’m a software developer! I can help you build a product , feature
+              or website Look through some of my work and experience! If you
+              like what you see and have a project you need coded, don’t
+              hestiate to contact me.
+            </p>
+          </div>
+          <!-- <div class="social-media-setting"> -->
+          <!-- <div class="social-media-title"> -->
+          <div class="lets-connect">
+            <a href="#"><b>LET’S CONNECT</b></a>
+          </div>
+          <!-- </div> -->
+          <!-- </div> -->
+          <div class="social-media-icons">
+            <a href="https://twitter.com/"
+              ><i class="tiwetter"><img src="img/t.png" alt="tiwetter" /></i
+            ></a>
+            <a href="https://www.linkedin.com/"
+              ><i class="linkedin"
+                ><img src="img/Linkedin.png" alt="linkedin" /></i
+            ></a>
+            <a href="https://medium.com/"
+              ><i class="medium"><img src="img/m.png" alt="Medium" /></i
+            ></a>
+            <a href="https://github.com/HussenYesufAlli"
+              ><i class="gt"><img src="img/g.png" alt="github" /></i
+            ></a>
+            <a href="#"
+              ><i class="fg"><img src="img/f.png" alt="f" /></i
+            ></a>
+          </div>
```

Comment:
[Required] Great job adding each image individually👍. However, as the previous reviewer suggested, please wrap the social icons in a `<ul>` tag instead of a `<div>` tag🙏. This is for semantic reasons, as ul tags are used to wrap most lists in a webpage

- In addition to that, please wrap each img in a `<li>` tag and make sure that the images are all links using an `<a>` tag. Your final code should look similar to something like this
`<ul>`
     `<li><a><img/></a></li>`
     `<li><a><img/></a></li>`
     `<li><a><img/></a></li>`
`</ul>`